### ISSUE TITLE
How to build?

### ISSUE THREAD

**Issue by andybak** [BenWoodford/SteamVR-Webkit#11]
I'm a bit of a C# noob and outside of Unity I'm a bit lost. I've tried building this in Visual Studio and I get as far as this error: 
    CS0246	The type or namespace name 'SteamVR_WebKit' could not be found (are you missing a using directive or an assembly reference?)

**Comment by BenWoodford** [comment]
Hm, that's strange and shouldn't happen. @robnewton did you have this problem?

**Comment by robnewton** [comment]
Nope, try deleting the VS solution and grabbing the latest version again.
